dashboards-observability/public/components/explorer/explorer.tsx

```diff
@@ -1250,6 +1130,8 @@ export const Explorer = ({
           tabId={tabId}
           baseQuery={appBaseQuery}
           stopLive={stopLive}
+          setIsLiveTailPopoverOpen={setIsLiveTailPopoverOpen}
```

Comment:
I'm thinking also like how about removing all these live tail related props and pass them directly to your live tail component, and then you pass the live tail component to search? Since all these props are not related to search component. Keep adding props that not related to search would make it unmanageable.

Reply:
i'll create another PR to include those changes